src/core/chat-commands.c

```diff
@@ -99,27 +99,27 @@ static SERVER_CONNECT_REC *get_server_connect(const char *data, int *plus_addr,
 	else if (g_hash_table_lookup(optlist, "4") != NULL)
 		conn->family = AF_INET;
 
-	if (g_hash_table_lookup(optlist, "ssl") != NULL)
-		conn->use_ssl = TRUE;
-	if ((tmp = g_hash_table_lookup(optlist, "ssl_cert")) != NULL)
-		conn->ssl_cert = g_strdup(tmp);
-	if ((tmp = g_hash_table_lookup(optlist, "ssl_pkey")) != NULL)
-		conn->ssl_pkey = g_strdup(tmp);
-	if ((tmp = g_hash_table_lookup(optlist, "ssl_pass")) != NULL)
-		conn->ssl_pass = g_strdup(tmp);
-	if (g_hash_table_lookup(optlist, "ssl_verify") != NULL)
-		conn->ssl_verify = TRUE;
-	if ((tmp = g_hash_table_lookup(optlist, "ssl_cafile")) != NULL)
-		conn->ssl_cafile = g_strdup(tmp);
-	if ((tmp = g_hash_table_lookup(optlist, "ssl_capath")) != NULL)
-		conn->ssl_capath = g_strdup(tmp);
-	if ((tmp = g_hash_table_lookup(optlist, "ssl_ciphers")) != NULL)
-		conn->ssl_ciphers = g_strdup(tmp);
-	if ((conn->ssl_capath != NULL && conn->ssl_capath[0] != '\0')
-	||  (conn->ssl_cafile != NULL && conn->ssl_cafile[0] != '\0'))
-		conn->ssl_verify = TRUE;
-	if ((conn->ssl_cert != NULL && conn->ssl_cert[0] != '\0') || conn->ssl_verify)
-		conn->use_ssl = TRUE;
+	if (g_hash_table_lookup(optlist, "tls") != NULL || g_hash_table_lookup(optlist, "ssl") != NULL)
+		conn->use_tls = TRUE;
+	if ((tmp = g_hash_table_lookup(optlist, "tls_cert")) != NULL || (tmp = g_hash_table_lookup(optlist, "ssl_cert")) != NULL)
+		conn->tls_cert = g_strdup(tmp);
+	if ((tmp = g_hash_table_lookup(optlist, "tls_pkey")) != NULL || (tmp = g_hash_table_lookup(optlist, "ssl_pkey")) != NULL)
+		conn->tls_pkey = g_strdup(tmp);
+	if ((tmp = g_hash_table_lookup(optlist, "tls_pass")) != NULL || (tmp = g_hash_table_lookup(optlist, "ssl_pass")) != NULL)
+		conn->tls_pass = g_strdup(tmp);
+	if (g_hash_table_lookup(optlist, "tls_verify") != NULL || g_hash_table_lookup(optlist, "ssl_verify") != NULL)
```

Comment:
So, if .irssi/config specifies ssl_verify (or tls_verify) = "no", it is != NULL and the connection requires verification...   This may cause confusion.  (caused me problems)

Reply:
Please don't use the code review features of pull requests for support questions, use the irc channel or github issues instead.